### ISSUE TITLE
Cannot install dev-bundle

### ISSUE THREAD

**Issue by zyzo** [meteor/meteor#3943]
When I try to run `scripts/generate-dev-bundle.sh`, the `dev_bundle` folder is not created and I got this error message :
```
[...]
abbrev@1.0.5 node_modules/abbrev
sigmund@1.0.0 node_modules/sigmund
combined-stream@0.0.7 node_modules/combined-stream
└── delayed-stream@0.0.5
lru-cache@2.5.0 node_modules/lru-cache
/tmp/generate-dev-bundle-YFBcZDYb/lib/node_modules/cordova/node_modules/cordova-lib/node_modules/cordova-js/node_modules/browserify/node_modules /tmp/generate-dev-bundle-YFBcZDYb/lib/node_modules
Missing (moved?): browserify-zlib/node_modules/pako/benchmark
```
I'm on Ubuntu 14.04, and have git, gcc, g++, autotools and scons installed

**Comment by stubailo** [comment]
You shouldn't have to generate the dev bundle yourself. You just need to run the `meteor` script in the root of the repo and it downloads one automatically for you.

**Comment by zyzo** [comment]
Thanks, that works. Am I missing something, or the README file contains  unclear instructions : 

> If you're the sort of person who likes to build everything from scratch, you can build all the Meteor dependencies (node.js, npm, mongodb, etc) with the provided script. This requires git, a C and C++ compiler, autotools, and scons. If you do not run this script, Meteor will automatically download pre-compiled binaries when you first run it.

> # OPTIONAL
> `./scripts/generate-dev-bundle.sh`

**Comment by glasser** [comment]
@stubailo is correct that you shouldn't have to do this, but the issue you reported was real and fixed in 3875e1aea833a8b4a6bb8d854cd27b644ab7bc7d.
